vhost-device-input/Cargo.toml

```diff
@@ -0,0 +1,36 @@
+[package]
+name = "vhost-device-input"
+version = "0.1.0"
+authors = ["Leo Yan <leo.yan@linaro.org>"]
+description = "vhost input backend device"
+repository = "https://github.com/rust-vmm/vhost-device"
+readme = "README.md"
+keywords = ["input", "vhost", "virt", "backend"]
+license = "Apache-2.0 OR BSD-3-Clause"
+edition = "2021"
+
+[features]
+xen = ["vm-memory/xen", "vhost/xen", "vhost-user-backend/xen"]
+
+[dependencies]
+clap = { version = "4.4",  features = ["derive"] }
+env_logger = "0.10"
+epoll = "4.3"
+libc = "0.2"
+log = "0.4"
+rand = "0.8.5"
+tempfile = "3.5"
+thiserror = "1.0"
+vhost = { version = "0.8", features = ["vhost-user-slave"] }
+vhost-user-backend = "0.10"
+virtio-bindings = "0.2.1"
+virtio-queue = "0.9"
+vm-memory = "0.12"
+vmm-sys-util = "0.11"
+evdev = { git = "https://github.com/emberian/evdev" }
+nix = "0.26"
```

Comment:
Does that play well with dependabot? The Cargo.lock file already locks this down, so it cannot update without getting bumped there. We probably want dependabot to remember us to bump to newer versions in order to see if there is any API breakage or such?

Reply:
Git revisions are not releases, so we can't rely on dependapot for this I guess. Eventually we need a published release which if doesn't exist we could ask the author to do.

Reply:
> Git revisions are not releases, so we can't rely on dependapot for this I guess. Eventually we need a published release which if doesn't exist we could ask the author to do.

Hm. I think it works as it is currently stated in the PR. It would get bumped by the cargo update that dependabot does? :thinking:

Reply:
What are you talking about? "I think it works as it is currently stated in the PR" where is it stated in the PR? What would get bumped by dependabot if your dependency is a specific git sha?

Reply:
> where is it stated in the PR?

It is stated as reference to the default branch of the repository.

> What would get bumped by dependabot if your dependency is a specific git sha?

The commit hash in the Cargo.lock file would be when running `cargo update`.

Reply:
Yes Eric, I suggested to add a specific SHA instead of the default branch ("Add a specific rev in evdev, so that we have no surprises in the future.").

Reply:
Yes, I understand that :). I was just arguing that the reference is already recorded in the .lock file and that adding it here might break dependabot - which I am not sure if that is what we want.